math/wide_integer/uintwide_t.h

```diff
@@ -2198,23 +2241,23 @@
     }
 
     template<const bool RePhraseIsSigned = IsSigned,
-             typename std::enable_if<(RePhraseIsSigned == false)>::type const* = nullptr>
-    static constexpr bool is_neg(uintwide_t<Width2, LimbType, AllocatorType, RePhraseIsSigned>)
+             typename std::enable_if<(!RePhraseIsSigned)>::type const* = nullptr>
+    static constexpr auto is_neg(uintwide_t<Width2, LimbType, AllocatorType, RePhraseIsSigned>) -> bool // NOLINT(hicpp-named-parameter,readability-named-parameter)
```

Comment:
Personal opinion: the Google C++ Style Guide is quite old and not universally accepted by any means. Some of the rules might deserve to be retired.

Alternatively, if you're excluding this because you got a different warning when you named the parameter, I've been there! It's vexing. But there's a get-out:

```c++
static constexpr auto is_neg(uintwide_t<Width2, LimbType, AllocatorType, RePhraseIsSigned> /*my_name*/) -> bool
```